src/index.js

```diff
@@ -8,4 +9,71 @@ const displayAllMovies = async () => {
   displayMovies(display);
 };
 
-window.addEventListener('load', displayAllMovies);
\ No newline at end of file
+window.addEventListener('load', displayAllMovies);
+
+const shows = await getMovies();
+const arr = Array.from(shows);
+
+const parentElem = document.getElementById('liContainer');
+parentElem.addEventListener('click', (event) => {
+  const matcher = event.target.matches('.btn');
+  const eventElem = event.target;
+  const eventId = eventElem.id - 1;
+
+  if (matcher) {
+    const newDiv = document.createElement('div');
+    newDiv.className = 'mainContainer';
+    newDiv.innerHTML = `
+    <div class="mainContainer1">
+
+    <div class="closeBtd flex">
+        <img id="closeImg" src="./images/close_icon.png" alt="main image">
+    </div>
+
+    <div class="movieDetails flex">
+        <div class="imgDiv">
```

Comment:
The mainContainer1 height is not enough to contain all items so it cause that the data information are overlap, it would be better to increase the height of moviesDetails container 👎 

<img width="146" alt="popup" src="https://github.com/Zohra-Neda/JavaScript-Capstone-Project/assets/58807251/2dc12674-fbf2-49e4-8d3c-4cc3878e67cd">

Reply:
resolved